pd/src/state.rs

```diff
@@ -152,15 +152,18 @@ INSERT INTO blobs (id, data) VALUES ('gc', $1)
         let app_hash = nct_anchor.to_bytes();
         let height = block.height.expect("height must be set");
 
+        // Any outputs in this block marked as quarantined will be delayed until this epoch index:
+        let unbonding_epoch_index =
+            block.epoch.as_ref().expect("block has epoch").index + block.unbonding_epochs;
+
```

Comment:
Not sure if I'm missing something, but is this currently being used?

Reply:
It will be shortly :)